mods/sethome/locale/sethome.ru.tr

```diff
@@ -6,3 +6,4 @@ Set a home using /sethome=Установите домашнюю точку, ис
 Set your home point=Установите вашу домашнюю точку
 Home set!=Домашняя точка установлена!
 Player not found!=Игрок не обнаружен!
+This command can be executed in-game only!=Эта команда может быть использована только в игре!
```

Comment:
This is one of the few strings I would *not* translate, as its purpose is to solely be visible *outside of the game*. Discord / IRC chat bridges don't handle translations.